MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,961 @@
+---
+layout: post
+title: Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI SfDataForm
+description: Learn here all about Migrating from Syncfusion Xamarin DataForm to Syncfusion .NET MAUI DataForm control and more.
```

Comment:
Learn here all about Migrating from Syncfusion Xamarin.Forms DataForm to Syncfusion .NET MAUI DataForm control.